src/components/Sections/services/formatServicesSections.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  IOCozyKonnector,
+  IOCozyTrigger,
+  IOCozyAccount
+} from 'cozy-client/types/types'
+
+import {
+  STATUS,
+  getAccountsFromTrigger,
+  getTriggersBySlug
+} from 'components/KonnectorHelpers'
+import { Section } from 'components/Sections/SectionsTypes'
+
+// List of categories that are allowed to be included even if pristine
+const categoriesWhitelist = [
```

Comment:
nit: Maybe we should extract this thing into a configuration file so that it's easier to find later 🤔